pkg/daemon/daemon.go

```diff
@@ -269,34 +259,13 @@ const (
 	sdMessageSessionStart = "8d45620c1a4348dbb17410da57c60c66"
 )
 
-func (dn *Daemon) detectEarlySSHAccessesFromBoot() error {
-	journal, err := sdjournal.NewJournal()
-	if err != nil {
-		return err
-	}
-	defer journal.Close()
-	if err := journal.AddMatch("MESSAGE_ID=" + sdMessageSessionStart); err != nil {
-		return err
-	}
-	if err := journal.SeekHead(); err != nil {
-		return err
-	}
-	r, err := journal.Next()
-	if err != nil {
-		return err
-	}
-	// journal EOF
-	if r == 0 {
-		return nil
-	}
-	// just one entry is enough to understand if someone jumped on the node
-	// from the very first boot
-	entry, err := journal.GetEntry()
+// detectThisBootSSHAccesses taints the node if we find a login before the daemon started up.
+func (dn *Daemon) detectThisBootSSHAccesses() error {
```

Comment:
could we use the old name `detectEarlySSHAccessesFromBoot` I think it reads clearer (and summarizes the func comment pretty well.